XenAdmin/Dialogs/ScheduledSnapshots/PolicyHistory.cs

```diff
@@ -30,17 +30,17 @@
  */
 
 using System;
-using System.Windows.Forms;
-using XenAdmin.Alerts;
-using XenAPI;
-using XenAdmin.Properties;
+using System.Collections.Generic;
 using System.ComponentModel;
+using System.Windows.Forms;
 using XenAdmin.Actions;
-using System.Collections.Generic;
+using XenAdmin.Alerts;
+using XenAdmin.Controls;
 using XenAdmin.Core;
+using XenAdmin.Properties;
+using XenAPI;
 
-
-namespace XenAdmin.Dialogs.VMProtectionRecovery
+namespace XenAdmin.Dialogs.ScheduledSnapshots

```

Comment:
Why do we need the pool property on line 47? It's assigned but we don't seem to do ahything with it.